script/release.sh

```diff
@@ -0,0 +1,292 @@
+  sed -i "s/$1/$2/g" README.md
+  
+  cd ..
+}
+
+changelog() {
+  cd $CLI
+  echo "Getting commit changes. Writing to ../changes.txt"
+  LOG=`git shortlog --email --no-merges --pretty=%s v${1}..`
+  echo -e "\`\`\`\n$LOG\n\`\`\`" > ../changes.txt
+  echo "Changelog has been written to changes.txt"
+  echo "!!PLEASE REVIEW BEFORE CONTINUING!!"
+  echo "Open changes.txt and add the release information"
+  echo "to the beginning of the file before the git shortlog"
+  cd ..
+}
+
+changelog_md() {
+  echo "Generating CHANGELOG.md"
+  CHANGES=$(cat changes.txt)
+  cd $CLI
+  DATE=$(date +"%m-%d-%Y")
+  CHANGELOG=$(cat CHANGELOG.md)
+  HEADER="## $CLI $1 ($DATE)"
+  echo -e "$HEADER\n\n$CHANGES\n\n$CHANGELOG" >CHANGELOG.md
+  echo "Changes have been written to CHANGELOG.md"
+  cd ..
+}
+
+git_commit() {
+  cd $CLI 
+
+  BRANCH=`git symbolic-ref --short HEAD`
+  if [ -z "$BRANCH" ]; then
+    echo "Unable to get branch name, is this even a git repo?"
+    return 1
+  fi
+  echo "Branch: " $BRANCH
+
+  git add .
+  git commit -m "$1 Release"
+  git push origin $BRANCH
+  hub pull-request -b $UPSTREAM_REPO/$CLI:master -h $ORIGIN_REPO/$CLI:$BRANCH
+
+  cd ..
+  echo ""
+  echo "PR opened against master to update version"
+  echo "MERGE THIS BEFORE CONTINUING"
+  echo ""
+}
+
+git_pull() {
+  cd $CLI
+  git pull
+  cd ..
+}
+
+sign() {
+  # Tarball it!
+  cp -r $CLI $CLI-$1
+  rm -rf $CLI-$1/.git*
+  tar czf $CLI-$1.tar.gz $CLI-$1
+  if [ $? -eq 0 ]; then
+        echo TARBALL OK
+  else
+        echo TARBALL FAIL
+        exit
+  fi
+
+  # Sign it!
+  echo -e "SIGN THE TARBALL!\n"
+  gpg --detach-sign --armor $CLI-$1.tar.gz
+  if [ $? -eq 0 ]; then
+        echo SIGN OK
+  else
+        echo SIGN FAIL
+        exit
+  fi
+
+  echo ""
+  echo "The tar.gz. is now located at $CLI-$1.tar.gz"
+  echo "and the signed one at $CLI-$1.tar.gz.asc"
+  echo ""
+}
+
+push() {
+  CHANGES=$(cat changes.txt)
+  # Release it!
+  github-release release \
+      --user $UPSTREAM_REPO \
+      --repo $CLI \
+      --tag v$1 \
+      --name "v$1" \
+      --description "$CHANGES"
+  if [ $? -eq 0 ]; then
+        echo RELEASE UPLOAD OK 
+  else 
+        echo RELEASE UPLOAD FAIL
+        exit
+  fi
+
+  github-release upload \
+      --user $UPSTREAM_REPO \
+      --repo $CLI \
+      --tag $1 \
+      --name "$CLI-$1.tar.gz" \
+      --file $CLI-$1.tar.gz
+  if [ $? -eq 0 ]; then
+        echo TARBALL UPLOAD OK 
+  else 
+        echo TARBALL UPLOAD FAIL
+        exit
+  fi
+
+  github-release upload \
+      --user $UPSTREAM_REPO \
+      --repo $CLI\
+      --tag $1 \
+      --name "$CLI-$1.tar.gz.asc" \
+      --file $CLI-$1.tar.gz.asc
+  if [ $? -eq 0 ]; then
+        echo SIGNED TARBALL UPLOAD OK 
+  else 
+        echo SIGNED TARBALL UPLOAD FAIL
+        exit
+  fi
+
+  echo "DONE"
+  echo "DOUBLE CHECK IT:"
+  echo "!!!"
+  echo "https://github.com/$UPSTREAM_REPO/$CLI/releases/edit/$1"
+  echo "!!!"
+  echo "REMEMBER TO UPDATE DOCKER BUILDS! :D"
```

Comment:
@sebgoa this was accidentally left in from a previous project, I'll remove this.